detection_rules/mappings.py

```diff
@@ -73,3 +77,80 @@ def get_rta_files(self, rta_list=None, rule_ids=None):
                 rta_files.add(rta_path)
 
         return list(sorted(rta_files))
+
+
+def get_triggered_rules() -> dict:
+    """Get the rules that are triggered by each RTA."""
+    triggered_rules = {}
+    for rta_test in get_available_tests().values():
+        for rule_info in rta_test.get("siem", []):
+            rule_id = rule_info.get("rule_id")
+            for platform in rta_test.get("platforms", []):
+                triggered_rules.setdefault(platform, []).append(rule_id)
+    return triggered_rules
+
+
+def get_platform_list(rule: TOMLRule) -> list:
+    """Get the list of OSes for a rule."""
+    os_list = []
+    if rule.contents.metadata.os_type_list:
```

Comment:
loaded rules will strip `None` values and lead to `AttributeError`s

```suggestion
    if rule.contents.metadata.get('os_type_list'):
```

Reply:
I think we can leave as-is. When I tested loaded the rules it didn't appear to strip None values. Does it happen in a specific use-case?

![Screen Shot 2022-09-07 at 11 09 07 AM](https://user-images.githubusercontent.com/1636709/188913579-43803df8-be12-42a1-9603-fb10c6ccf488.png)

Reply:
ah right, it strips on serialization, so this should be fine